setup.cfg

```diff
@@ -0,0 +1,236 @@
+[metadata]
+name = farm-haystack
+version = file: VERSION.txt
+url = https://github.com/deepset-ai/haystack
+project_urls =
+    Docs: RTD = https://haystack.deepset.ai/overview/intro
+    CI: GitHub = https://github.com/deepset-ai/haystack/actions
+    GitHub: issues = https://github.com/deepset-ai/haystack/issues
+    GitHub: repo = https://github.com/deepset-ai/haystack
+description = Neural Question Answering & Semantic Search at Scale. Use modern transformer based models like BERT to find answers in large document collections
+long_description = file: README.md
+long_description_content_type = text/markdown
+keywords=
+    QA 
+    Question-Answering 
+    Reader 
+    Retriever 
+    semantic-search 
+    search 
+    BERT 
+    roberta 
+    albert 
+    squad 
+    mrc 
+    transfer-learning 
+    language-model 
+    transformer
+author = deepset.ai
+author_email = malte.pietsch@deepset.ai
+license = Apache License 2.0
+license_file = LICENSE
+platforms = any
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Science/Research
+    License :: Freely Distributable
+    License :: OSI Approved :: Apache Software License
+    Topic :: Scientific/Engineering :: Artificial Intelligence
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+
+[options]
+use_scm_version = True
+python_requires = >=3.7
+packages = find:
+install_requires =
+    importlib-metadata; python_version < '3.8'
+    torch>1.9,<1.11
+    requests
+    pydantic
+    transformers==4.13.0
+    nltk
+    pandas
+
+    # Utils
+    dill  # pickle extension for (de-)serialization
+    tqdm  # progress bars in model download and training scripts
+    networkx  # graphs library
+    mmh3  # fast hashing function (murmurhash3)
+    quantulum3  # quantity extraction from text
+    azure-ai-formrecognizer==3.2.0b2  # forms reader
+
+    # Preprocessing
+    more_itertools  # for windowing
+    python-docx
+    langdetect   # for PDF conversions
+    tika  # Apache Tika (text & metadata extractor)
+
+    # See haystack/nodes/retriever/_embedding_encoder.py, _SentenceTransformersEmbeddingEncoder
+    sentence-transformers>=0.4.0
+
+    # for stats in run_classifier
+    scipy>=1.3.2  
+    scikit-learn>=1.0.0
+
+    # Metrics and logging
+    seqeval
+    mlflow<=1.13.1 
+
+    # Elasticsearch
+    elasticsearch>=7.7,<=7.10
+    elastic-apm
+
+    # Not found in repo, to review:
+    #httptools
+    #dataclasses-json
+    #uvloop==0.14; sys_platform != 'win32' and sys_platform != 'cygwin'
```

Comment:
Just that we do not forget that ;-)

Reply:
I found no trace of usage in the repo and all tests pass without them... I might trigger the Windows CI to check for failures, but if it also passes I'll remove them. What do you think?

At least `dataclasses-json` I'm quite sure it's a leftover from the primitives refactoring from @tholor. The others I have no clue